File: tests/fixtures/edgecase_repo/python/pkg/module8.py
"""Module 8 of the generated Python layer."""

from __future__ import annotations

from pkg.module7 import Service7


class Base8:
    """Base class for service 8."""


class Service8(Base8):
    """Service 8."""

    def handle(self, payload):
        """Handle a payload."""
        self.validate(payload)
        return Service7().handle(payload)

    def validate(self, payload):
        """Validate a payload."""
        return payload is not None


def helper_8(value):
    """Module-level helper."""
    return value

File: tests/fixtures/edgecase_repo/go/module8/module8.go
package module8

import (
	"fmt"

	"example.com/pkg/module7"
)

type Base8 struct {
	Name string
}

type Service8 struct {
	Base8
}

type Handler8 interface {
	Handle(payload string) bool
}

func (s *Service8) Handle(payload string) bool {
	s.validate(payload)
	fmt.Println(payload)
	return true
}

func (s *Service8) validate(payload string) {
}

func Helper8(value string) string {
	_ = module7.Helper7("x")
	return value
}
